src/lbaf/Applications/conf.yaml

```diff
@@ -78,16 +79,17 @@ output_dir: ../../../output
 output_file_stem: output_file
 
 # Simulate from samplers (comment out if reading from log)
-#n_objects: 200
-#n_mapped_ranks: 2
-#communication_degree: 20
-#time_sampler:
-#  name: lognormal
-#  parameters:
-#    - 1.0
-#    - 10.0
-#volume_sampler:
-#  name: lognormal
-#  parameters:
-#    - 1.0
-#    - 10.0
+#from_samplers:
```

Comment:
good idea but make sure it's either/or: if both are turned ON or both are turned OFF ==> ERROR

Reply:
it is defined already, I am working now on the tests, to check that 100%